websites/P/Pulse/dist/metadata.json

```diff
@@ -1,22 +1,29 @@
 {
+  "$schema": "https://schemas.premid.app/metadata/1.0",
   "author": {
     "name": "Callum",
     "id": "506899274748133376"
   },
+  "contributors": [
+    {
+      "name": "Ben",
+      "id": "200636868621107200"
+    }
+  ],
   "url": "itspulse.net",
   "description": {
     "en": "Pulse is an internet radio community originally started in 2019, after much disrespect and un-professional antics across the radio community; we decided to make our own project that gives people the chance and opportunity to take control of how they want a community."
   },
   "service": "Pulse",
-  "version": "1.2.4",
-  "logo": "https://i.imgur.com/mqYHGOC.png",
-  "thumbnail": "https://i.imgur.com/a7dST50.png",
+  "version": "2.0.0",
+  "logo": "https://i.imgur.com/IPlZPxB.png",
+  "thumbnail": "https://i.imgur.com/IPlZPxB.png",
```

Comment:
thumbnail should be banner of this site or screenshot. This is valid one https://i.imgur.com/a7dST50.png